examples/segmentation/train_deeplabv3_ade20k.py

```diff
@@ -328,6 +330,9 @@ def combo_loss(output, target):
         logger = WandBLogger(entity=args.wandb_entity, project=args.wandb_project)
         logging.info('Built Weights and Biases logger')
 
+    callbacks = [speed_monitor, lr_monitor, checkpoint_saver]
+    if args.image_viz:
+        callbacks.append(ImageVisualizer())
```

Comment:
I guess its ok to just use default args here, but should probably set mode to segmentation?

Reply:
fixed